### PR TITLE
Various CCPP PRs. Improve and update CI.

### PR DESCRIPTION
**Pull request by MinsukJi-NOAA**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] If new or updated input data is required by this PR, it is clearly stated in the text of the PR.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

This PR includes:
* Various CCPP PRs as described in https://github.com/NOAA-EMC/fv3atm/pull/291
* Improvement and update for CI

### Issue(s) addressed

https://github.com/NCAR/ccpp-physics/pull/641
#539

## Testing

Regression testing on tier-1 platforms using the newly generated baselines:

- [x] hera.intel
- [x] hera.gnu
- [ ] orion.intel - note: skip for this PR due to maintenance
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3

Testing of the actions workflow runs can be seen [here](https://github.com/MinsukJi-NOAA/ufs-weather-model/actions/runs/775414486) and [here](https://github.com/MinsukJi-NOAA/ufs-weather-model/actions/runs/775414575).

Note also that @climbfuji compared the new baselines (20210427) on Hera for Intel and GNU to the previous ones (20210426) for the final PR. Only the tests described in https://github.com/NOAA-EMC/fv3atm/pull/291#issue-622373161 are different, as expected.

## Dependencies

- waiting on https://github.com/NOAA-EMC/fv3atm/pull/291

### CODE REVIEW

**Comment by BrianCurtis-NOAA** [comment]
Are you doing something unique to check for `run-ci` in the commit message? Would using something like 
`if: "contains(github.event.head_commit.message, 'run-ci')"` 
only run the jobs if they see the commit message with `run-ci` ?

**Comment by MinsukJi-NOAA** [comment]
> Are you doing something unique to check for `run-ci` in the commit message? Would using something like
> `if: "contains(github.event.head_commit.message, 'run-ci')"`
> only run the jobs if they see the commit message with `run-ci` ?

`github.event.head_commit.message` context is only available for push event, and not for pull_request event. I guess I can maybe simplify the push part of the if-elif-fi statement. I will check.

**Comment by aerorahul** [comment]
a quick glance at the python script:
- standard indents are 4 spaces, not 2.
- this script will fail coding norms when using `pycodestyle`
- `autopep8 -i script.py` will fix all coding norm issues.

**Comment by aerorahul** [comment]
I see unlimited `while` loops in the python script.   What happens if the it never completes or fails to return a valid response?

**Comment by MinsukJi-NOAA** [comment]
> I see unlimited `while` loops in the python script. What happens if the it never completes or fails to return a valid response?

CI job will have to be manually canceled on a case-by-case basis. It is hard to put a time limit that will automatically stop the loops because it all depends on the number and status of jobs in the queue and in progress. If not manually stopped, it will be cut off by the time limit of an actions job, which is 6 hours by default. I guess I could put in the six hours in the script. If you have a suggestion, please let me know.

**Comment by aerorahul** [comment]
> > I see unlimited `while` loops in the python script. What happens if the it never completes or fails to return a valid response?
> 
> CI job will have to be manually canceled on a case-by-case basis. It is hard to put a time limit that will automatically stop the loops because it all depends on the number and status of jobs in the queue and in progress. If you know of a way, please let me know.

I don't know at the moment, but I have not thought about this.
It does however defeat the purpose of a CI/CD system if manual intervention is necessary.
I can think on it, but go ahead with the PR.

**Comment by MinsukJi-NOAA** [comment]
> 
> I don't know at the moment, but I have not thought about this.
> It does however defeat the purpose of a CI/CD system if manual intervention is necessary.
> I can think on it, but go ahead with the PR.

I agree it's not a perfect system. I have some ideas, and they will be experimented with, tested, and implemented in the future.

**Comment by MinsukJi-NOAA** [comment]
@DusanJovic-NOAA , @BrianCurtis-NOAA, @aerorahul , could you please help me conduct one more testing by forking my repository (MinsukJi-NOAA:feature/CI-update) and making 2 pull requests per person? 1st pull request with run-ci in the commit message, and 2nd pull request without run-ci.  Thanks.

**Comment by aerorahul** [comment]
@MinsukJi-NOAA 
Have you looked at some of these actions?
https://github.com/marketplace/actions/wait-for-check

**Comment by MinsukJi-NOAA** [comment]
Thanks for testing @DusanJovic-NOAA. It was successful: https://github.com/MinsukJi-NOAA/ufs-weather-model/actions

**Comment by MinsukJi-NOAA** [comment]
This PR will not pass the CI test because it contains changes for the CI test scripts.

**Comment by BrianCurtis-NOAA** [comment]
Machine: gaea
Compiler: intel
Job: BL
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/621511026/20210427134507/ufs-weather-model
Please make changes and add the following label back:
gaea-intel-BL

**Comment by climbfuji** [comment]
auto-bl failed on gaea. Looks like a filesystem problem (timed out at the very beginning of the cmake call for all compile jobs). Will try again manually.

**Comment by BrianCurtis-NOAA** [comment]
Machine: jet
Compiler: intel
Job: RT
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/621511026/20210427134511/ufs-weather-model
Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_295493
Test fv3_multigases 016 failed in check_result failed 
Test fv3_multigases 016 failed in run_test failed 
Test fv3_lndp 013 failed in check_result failed 
Test fv3_lndp 013 failed in run_test failed 
Test fv3_control 001 failed in check_result failed 
Test fv3_control 001 failed in run_test failed 
Test fv3_wrtGauss_nemsio_c192 010 failed in check_result failed 
Test fv3_wrtGauss_nemsio_c192 010 failed in run_test failed 
Test fv3_wrtGauss_nemsio 009 failed in check_result failed 
Test fv3_wrtGauss_nemsio 009 failed in run_test failed 
Test fv3_wrtGlatlon_netcdf 008 failed in check_result failed 
Test fv3_wrtGlatlon_netcdf 008 failed in run_test failed 
Test fv3_wrtGauss_netcdf_esmf 005 failed in check_result failed 
Test fv3_wrtGauss_netcdf_esmf 005 failed in run_test failed 
Test fv3_wrtGauss_netcdf 006 failed in check_result failed 
Test fv3_wrtGauss_netcdf 006 failed in run_test failed 
Test fv3_lheatstrg 015 failed in check_result failed 
Test fv3_lheatstrg 015 failed in run_test failed 
Test fv3_stretched_nest 019 failed in check_result failed 
Test fv3_stretched_nest 019 failed in run_test failed 
Test fv3_wrtGauss_netcdf_parallel 007 failed in check_result failed 
Test fv3_wrtGauss_netcdf_parallel 007 failed in run_test failed 
Test fv3_ca 012 failed in check_result failed 
Test fv3_ca 012 failed in run_test failed 
Test fv3_control_32bit 017 failed in check_result failed 
Test fv3_control_32bit 017 failed in run_test failed 
Test fv3_stochy 011 failed in check_result failed 
Test fv3_stochy 011 failed in run_test failed 
Test fv3_gfdlmp 025 failed in check_result failed 
Test fv3_gfdlmp 025 failed in run_test failed 
Test fv3_gfdlmprad_gwd 026 failed in check_result failed 
Test fv3_gfdlmprad_gwd 026 failed in run_test failed 
Test fv3_gfdlmprad_noahmp 027 failed in check_result failed 
Test fv3_gfdlmprad_noahmp 027 failed in run_test failed 
Test fv3_2threads 002 failed in check_result failed 
Test fv3_2threads 002 failed in run_test failed 
Test fv3_regional_quilt_netcdf_parallel 024 failed in check_result failed 
Test fv3_regional_quilt_netcdf_parallel 024 failed in run_test failed 
Test fv3_regional_quilt_hafs 023 failed in check_result failed 
Test fv3_regional_quilt_hafs 023 failed in run_test failed 
Test fv3_regional_quilt 022 failed in check_result failed 
Test fv3_regional_quilt 022 failed in run_test failed 
Test fv3_regional_control 020 failed in check_result failed 
Test fv3_regional_control 020 failed in run_test failed 
Test fv3_stretched 018 failed in check_result failed 
Test fv3_stretched 018 failed in run_test failed 
Test fv3_satmedmfq 030 failed in check_result failed 
Test fv3_satmedmfq 030 failed in run_test failed 
Test fv3_satmedmf 029 failed in check_result failed 
Test fv3_satmedmf 029 failed in run_test failed 
Test fv3_csawmg 028 failed in check_result failed 
Test fv3_csawmg 028 failed in run_test failed 
Test fv3_gfs_v16_flake 047 failed in check_result failed 
Test fv3_gfs_v16_flake 047 failed in run_test failed 
Test fv3_gocart_clm 046 failed in check_result failed 
Test fv3_gocart_clm 046 failed in run_test failed 
Test fv3_gfs_v16_stochy 040 failed in check_result failed 
Test fv3_gfs_v16_stochy 040 failed in run_test failed 
Test fv3_gfsv16_csawmgt 045 failed in check_result failed 
Test fv3_gfsv16_csawmgt 045 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP 041 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP 041 failed in run_test failed 
Test fv3_gfs_v16 038 failed in check_result failed 
Test fv3_gfs_v16 038 failed in run_test failed 
Test fv3_gfsv16_csawmg 044 failed in check_result failed 
Test fv3_gfsv16_csawmg 044 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_2thrd 043 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_2thrd 043 failed in run_test failed 
Test fv3_gfdlmp_32bit 031 failed in check_result failed 
Test fv3_gfdlmp_32bit 031 failed in run_test failed 
Test fv3_gfdlmprad_32bit_post 032 failed in check_result failed 
Test fv3_gfdlmprad_32bit_post 032 failed in run_test failed 
Test fv3_rrfs_v1alpha 035 failed in check_result failed 
Test fv3_rrfs_v1alpha 035 failed in run_test failed 
Test fv3_thompson_no_aero 037 failed in check_result failed 
Test fv3_thompson_no_aero 037 failed in run_test failed 
Test fv3_thompson 036 failed in check_result failed 
Test fv3_thompson 036 failed in run_test failed 
Test fv3_gsd 034 failed in check_result failed 
Test fv3_gsd 034 failed in run_test failed 
Test fv3_cpt 033 failed in check_result failed 
Test fv3_cpt 033 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_c192L127 042 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_c192L127 042 failed in run_test failed 
Test fv3_gfs_v16_debug 053 failed in check_result failed 
Test fv3_gfs_v16_debug 053 failed in run_test failed 
Test fv3_gfs_v16_RRTMGP_debug 054 failed in check_result failed 
Test fv3_gfs_v16_RRTMGP_debug 054 failed in run_test failed 
Test fv3_control_debug 056 failed in check_result failed 
Test fv3_control_debug 056 failed in run_test failed 
Test fv3_gsd_diag3d_debug 059 failed in check_result failed 
Test fv3_gsd_diag3d_debug 059 failed in run_test failed 
Test fv3_thompson_no_aero_debug 061 failed in check_result failed 
Test fv3_thompson_no_aero_debug 061 failed in run_test failed 
Test fv3_gsd_debug 058 failed in check_result failed 
Test fv3_gsd_debug 058 failed in run_test failed 
Test fv3_stretched_nest_debug 057 failed in check_result failed 
Test fv3_stretched_nest_debug 057 failed in run_test failed 
Test fv3_rrfs_v1beta_debug 062 failed in check_result failed 
Test fv3_rrfs_v1beta_debug 062 failed in run_test failed 
Test fv3_thompson_debug 060 failed in check_result failed 
Test fv3_thompson_debug 060 failed in run_test failed 
Test fv3_regional_control_debug 055 failed in check_result failed 
Test fv3_regional_control_debug 055 failed in run_test failed 
Test fv3_rrfs_v1alpha_debug 063 failed in check_result failed 
Test fv3_rrfs_v1alpha_debug 063 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 048 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson 048 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 064 failed in check_result failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 064 failed in run_test failed 
Test fv3_gfsv16_ugwpv1 050 failed in check_result failed 
Test fv3_gfsv16_ugwpv1 050 failed in run_test failed 
Test fv3_gfs_v16_ras_debug 067 failed in check_result failed 
Test fv3_gfs_v16_ras_debug 067 failed in run_test failed 
Test fv3_gfs_v16_ras 052 failed in check_result failed 
Test fv3_gfs_v16_ras 052 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 065 failed in check_result failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 065 failed in run_test failed 
Test fv3_gfsv16_ugwpv1_warmstart 051 failed in check_result failed 
Test fv3_gfsv16_ugwpv1_warmstart 051 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 049 failed in check_result failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 049 failed in run_test failed 
Test fv3_gfsv16_ugwpv1_debug 066 failed in check_result failed 
Test fv3_gfsv16_ugwpv1_debug 066 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-RT

**Comment by climbfuji** [comment]
> Machine: jet
> Compiler: intel
> Job: RT
> Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/621511026/20210427134511/ufs-weather-model
> Please manually delete: /lfs4/HFIP/h-nems/emc.nemspara/RT_RUNDIRS/emc.nemspara/FV3_RT/rt_295493
> Test fv3_multigases 016 failed in check_result failed
> Test fv3_multigases 016 failed in run_test failed
> Test fv3_lndp 013 failed in check_result failed
> Test fv3_lndp 013 failed in run_test failed
> Test fv3_control 001 failed in check_result failed
> Test fv3_control 001 failed in run_test failed
> Test fv3_wrtGauss_nemsio_c192 010 failed in check_result failed
> Test fv3_wrtGauss_nemsio_c192 010 failed in run_test failed
> Test fv3_wrtGauss_nemsio 009 failed in check_result failed
> Test fv3_wrtGauss_nemsio 009 failed in run_test failed
> Test fv3_wrtGlatlon_netcdf 008 failed in check_result failed
> Test fv3_wrtGlatlon_netcdf 008 failed in run_test failed
> Test fv3_wrtGauss_netcdf_esmf 005 failed in check_result failed
> Test fv3_wrtGauss_netcdf_esmf 005 failed in run_test failed
> Test fv3_wrtGauss_netcdf 006 failed in check_result failed
> Test fv3_wrtGauss_netcdf 006 failed in run_test failed
> Test fv3_lheatstrg 015 failed in check_result failed
> Test fv3_lheatstrg 015 failed in run_test failed
> Test fv3_stretched_nest 019 failed in check_result failed
> Test fv3_stretched_nest 019 failed in run_test failed
> Test fv3_wrtGauss_netcdf_parallel 007 failed in check_result failed
> Test fv3_wrtGauss_netcdf_parallel 007 failed in run_test failed
> Test fv3_ca 012 failed in check_result failed
> Test fv3_ca 012 failed in run_test failed
> Test fv3_control_32bit 017 failed in check_result failed
> Test fv3_control_32bit 017 failed in run_test failed
> Test fv3_stochy 011 failed in check_result failed
> Test fv3_stochy 011 failed in run_test failed
> Test fv3_gfdlmp 025 failed in check_result failed
> Test fv3_gfdlmp 025 failed in run_test failed
> Test fv3_gfdlmprad_gwd 026 failed in check_result failed
> Test fv3_gfdlmprad_gwd 026 failed in run_test failed
> Test fv3_gfdlmprad_noahmp 027 failed in check_result failed
> Test fv3_gfdlmprad_noahmp 027 failed in run_test failed
> Test fv3_2threads 002 failed in check_result failed
> Test fv3_2threads 002 failed in run_test failed
> Test fv3_regional_quilt_netcdf_parallel 024 failed in check_result failed
> Test fv3_regional_quilt_netcdf_parallel 024 failed in run_test failed
> Test fv3_regional_quilt_hafs 023 failed in check_result failed
> Test fv3_regional_quilt_hafs 023 failed in run_test failed
> Test fv3_regional_quilt 022 failed in check_result failed
> Test fv3_regional_quilt 022 failed in run_test failed
> Test fv3_regional_control 020 failed in check_result failed
> Test fv3_regional_control 020 failed in run_test failed
> Test fv3_stretched 018 failed in check_result failed
> Test fv3_stretched 018 failed in run_test failed
> Test fv3_satmedmfq 030 failed in check_result failed
> Test fv3_satmedmfq 030 failed in run_test failed
> Test fv3_satmedmf 029 failed in check_result failed
> Test fv3_satmedmf 029 failed in run_test failed
> Test fv3_csawmg 028 failed in check_result failed
> Test fv3_csawmg 028 failed in run_test failed
> Test fv3_gfs_v16_flake 047 failed in check_result failed
> Test fv3_gfs_v16_flake 047 failed in run_test failed
> Test fv3_gocart_clm 046 failed in check_result failed
> Test fv3_gocart_clm 046 failed in run_test failed
> Test fv3_gfs_v16_stochy 040 failed in check_result failed
> Test fv3_gfs_v16_stochy 040 failed in run_test failed
> Test fv3_gfsv16_csawmgt 045 failed in check_result failed
> Test fv3_gfsv16_csawmgt 045 failed in run_test failed
> Test fv3_gfs_v16_RRTMGP 041 failed in check_result failed
> Test fv3_gfs_v16_RRTMGP 041 failed in run_test failed
> Test fv3_gfs_v16 038 failed in check_result failed
> Test fv3_gfs_v16 038 failed in run_test failed
> Test fv3_gfsv16_csawmg 044 failed in check_result failed
> Test fv3_gfsv16_csawmg 044 failed in run_test failed
> Test fv3_gfs_v16_RRTMGP_2thrd 043 failed in check_result failed
> Test fv3_gfs_v16_RRTMGP_2thrd 043 failed in run_test failed
> Test fv3_gfdlmp_32bit 031 failed in check_result failed
> Test fv3_gfdlmp_32bit 031 failed in run_test failed
> Test fv3_gfdlmprad_32bit_post 032 failed in check_result failed
> Test fv3_gfdlmprad_32bit_post 032 failed in run_test failed
> Test fv3_rrfs_v1alpha 035 failed in check_result failed
> Test fv3_rrfs_v1alpha 035 failed in run_test failed
> Test fv3_thompson_no_aero 037 failed in check_result failed
> Test fv3_thompson_no_aero 037 failed in run_test failed
> Test fv3_thompson 036 failed in check_result failed
> Test fv3_thompson 036 failed in run_test failed
> Test fv3_gsd 034 failed in check_result failed
> Test fv3_gsd 034 failed in run_test failed
> Test fv3_cpt 033 failed in check_result failed
> Test fv3_cpt 033 failed in run_test failed
> Test fv3_gfs_v16_RRTMGP_c192L127 042 failed in check_result failed
> Test fv3_gfs_v16_RRTMGP_c192L127 042 failed in run_test failed
> Test fv3_gfs_v16_debug 053 failed in check_result failed
> Test fv3_gfs_v16_debug 053 failed in run_test failed
> Test fv3_gfs_v16_RRTMGP_debug 054 failed in check_result failed
> Test fv3_gfs_v16_RRTMGP_debug 054 failed in run_test failed
> Test fv3_control_debug 056 failed in check_result failed
> Test fv3_control_debug 056 failed in run_test failed
> Test fv3_gsd_diag3d_debug 059 failed in check_result failed
> Test fv3_gsd_diag3d_debug 059 failed in run_test failed
> Test fv3_thompson_no_aero_debug 061 failed in check_result failed
> Test fv3_thompson_no_aero_debug 061 failed in run_test failed
> Test fv3_gsd_debug 058 failed in check_result failed
> Test fv3_gsd_debug 058 failed in run_test failed
> Test fv3_stretched_nest_debug 057 failed in check_result failed
> Test fv3_stretched_nest_debug 057 failed in run_test failed
> Test fv3_rrfs_v1beta_debug 062 failed in check_result failed
> Test fv3_rrfs_v1beta_debug 062 failed in run_test failed
> Test fv3_thompson_debug 060 failed in check_result failed
> Test fv3_thompson_debug 060 failed in run_test failed
> Test fv3_regional_control_debug 055 failed in check_result failed
> Test fv3_regional_control_debug 055 failed in run_test failed
> Test fv3_rrfs_v1alpha_debug 063 failed in check_result failed
> Test fv3_rrfs_v1alpha_debug 063 failed in run_test failed
> Test fv3_HAFS_v0_hwrf_thompson 048 failed in check_result failed
> Test fv3_HAFS_v0_hwrf_thompson 048 failed in run_test failed
> Test fv3_HAFS_v0_hwrf_thompson_debug 064 failed in check_result failed
> Test fv3_HAFS_v0_hwrf_thompson_debug 064 failed in run_test failed
> Test fv3_gfsv16_ugwpv1 050 failed in check_result failed
> Test fv3_gfsv16_ugwpv1 050 failed in run_test failed
> Test fv3_gfs_v16_ras_debug 067 failed in check_result failed
> Test fv3_gfs_v16_ras_debug 067 failed in run_test failed
> Test fv3_gfs_v16_ras 052 failed in check_result failed
> Test fv3_gfs_v16_ras 052 failed in run_test failed
> Test fv3_esg_HAFS_v0_hwrf_thompson_debug 065 failed in check_result failed
> Test fv3_esg_HAFS_v0_hwrf_thompson_debug 065 failed in run_test failed
> Test fv3_gfsv16_ugwpv1_warmstart 051 failed in check_result failed
> Test fv3_gfsv16_ugwpv1_warmstart 051 failed in run_test failed
> Test fv3_esg_HAFS_v0_hwrf_thompson 049 failed in check_result failed
> Test fv3_esg_HAFS_v0_hwrf_thompson 049 failed in run_test failed
> Test fv3_gfsv16_ugwpv1_debug 066 failed in check_result failed
> Test fv3_gfsv16_ugwpv1_debug 066 failed in run_test failed
> Please make changes and add the following label back:
> jet-intel-RT

My mistake. I added the jet-intel-rt label instead of the -bl label. Trying again.

**Comment by BrianCurtis-NOAA** [comment]
> This PR will not pass the CI test because it contains changes for the CI test scripts.

Im just curious, if the CI changes are in this PR, does it not use these files when it launches the CI?

**Comment by MinsukJi-NOAA** [comment]
> > This PR will not pass the CI test because it contains changes for the CI test scripts.
> 
> Im just curious, if the CI changes are in this PR, does it not use these files when it launches the CI?

The workflow triggered by pull_request (build_test.yml) will use the new file. However, the workflow it triggers (aux.yml) uses the file in the base repository. This is for security reasons as it has the write permission to the repository as well as access to secrets. The CI test does not work because the two workflows work in tandem.

**Comment by BrianCurtis-NOAA** [comment]
> > > This PR will not pass the CI test because it contains changes for the CI test scripts.
> > 
> > 
> > Im just curious, if the CI changes are in this PR, does it not use these files when it launches the CI?
> 
> The workflow triggered by pull_request (build_test.yml) will use the new file. However, the workflow it triggers (aux.yml) uses the file in the base repository. This is for security reasons as it has the write permission to the repository as well as access to secrets. The CI test does not work because the two workflows work in tandem.

Thanks for the info. Much appreciated.

**Comment by climbfuji** [comment]
Since the CI tests cannot pass as @MinsukJi-NOAA explained and all regression tests except orion.intel passed (which was under maintenance for most of the day), we can consider these PRs as ready to merge. I did start auto-bl on orion, and given how late it is we can just wait to tomorrow = Tuesday morning to do the merges.

**Comment by BrianCurtis-NOAA** [comment]
Machine: orion
Compiler: intel
Job: BL
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/621511026/20210427194528/ufs-weather-model
Please make changes and add the following label back:
orion-intel-BL

**Comment by climbfuji** [comment]
> Machine: orion
> Compiler: intel
> Job: BL
> Repo location: /work/noaa/nems/emc.nemspara/autort/pr/621511026/20210427194528/ufs-weather-model
> Please make changes and add the following label back:
> orion-intel-BL

I checked the orion log, one test failed while creating the baseline due to an MPI timeout (a system issue as so often on orion). Let's proceed with the merge, in the meanwhile I will try to complete this manually on orion.

**Comment by MinsukJi-NOAA** [comment]
CCPP pointers are not updated. Let me try again.

**Review by junwang-noaa — APPROVED** [review]

**Comment by climbfuji** [comment]
> CCPP pointers are not updated. Let me try again.

You need to update your fv3atm submodule, it needs to point to hash 28888f072c9a38b7d8e3d2df864d822e7f489733 in the NOAA-EMC repository.

**Comment by MinsukJi-NOAA** [comment]
OK. It's done and ready for merge.

**Review by climbfuji — APPROVED** [review]
fv3atm submodule pointer is correct

**Comment by climbfuji** [comment]
@DusanJovic-NOAA please review and merge if ok - thank you!

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Comment by climbfuji** [comment]
Note. I was able to create the 20210427 baseline for this PR on Orion retrospectively and I started to verify against it, but then the `/work` filesystem died again. At least the baseline is there (if it is ever needed).
